### DISCUSSION TITLE
automerge is not merging patch PRs

### DISCUSSION THREAD

**Discussion by blacelle** [renovatebot/renovate#13106] [Request Help] [answered]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Please select which platform you are using if self-hosting.

github.com

### Describe the bug

I consider the following configuration:

```
{
  "extends": [
    "config:base"
  ],
  "labels": ["renovate"],
  "schedule": "every weekend",
  "packageRules": [
    {
      "matchUpdateTypes": ["patch", "pin", "digest"],
      "automerge": true
    },
    {
      "matchDepTypes": ["devDependencies"],
      "automerge": true
    }
  ]
}

```

Renovate opens a PR like:

Package | Update | Change
-- | -- | --
amazon/opendistro-for-elasticsearch | patch | 1.13.2 -> 1.13.3

I would expect the PR to be merged automatically, but it remains open. In logs, I see:

...

I suppose automerge does not kicks-in due to: 

> DEBUG: Branch status yellow(branch="renovate/amazon-opendistro-for-elasticsearch-1.x")

However, the PR now looks OK, still it is not automerged:
![image](https://user-images.githubusercontent.com/2117911/146009306-c6bd74df-ec83-46cf-b04b-8aa22cbe5775.png)

Is there something wrong in my configuration?

Thanks

My own references: (private repo)
https://app.renovatebot.com/dashboard#github/solven-eu/mitrust-datasharing/529570272
https://github.com/solven-eu/mitrust-datasharing/pull/10185

### Relevant debug logs

<details><summary>Logs</summary>

```

DEBUG: Adding labels 'renovate' to #10185(branch="renovate/amazon-opendistro-for-elasticsearch-1.x")
INFO: PR created(branch="renovate/amazon-opendistro-for-elasticsearch-1.x")
{
  "pr": 10185,
  "prTitle": "Update amazon/opendistro-for-elasticsearch Docker tag to v1.13.3"
}
DEBUG: resolveBranchStatus(branchName=renovate/amazon-opendistro-for-elasticsearch-1.x, ignoreTests=false)(branch="renovate/amazon-opendistro-for-elasticsearch-1.x")
DEBUG: getBranchStatus(renovate/amazon-opendistro-for-elasticsearch-1.x)(branch="renovate/amazon-opendistro-for-elasticsearch-1.x")
DEBUG: branch status check result(branch="renovate/amazon-opendistro-for-elasticsearch-1.x")
{
  "state": "pending",
  "statuses": []
}
DEBUG: No check runs found(branch="renovate/amazon-opendistro-for-elasticsearch-1.x")
{
  "result": {
    "total_count": 0,
    "check_runs": []
  }
}
DEBUG: Branch status yellow(branch="renovate/amazon-opendistro-for-elasticsearch-1.x")
DEBUG: Branch status is: yellow(branch="renovate/amazon-opendistro-for-elasticsearch-1.x")
DEBUG: Skipping assignees and reviewers as automerge=true(branch="renovate/amazon-opendistro-for-elasticsearch-1.x")
DEBUG: Created Pull Request #10185(branch="renovate/amazon-opendistro-for-elasticsearch-1.x")
DEBUG: PR is configured for automerge(branch="renovate/amazon-opendistro-for-elasticsearch-1.x")
DEBUG: PR is not ready for merge(branch="renovate/amazon-opendistro-for-elasticsearch-1.x")
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/6311784/146014470-aed02a95-43ba-4600-ba72-f8d4f814f5c3.png)

**Reply by rarkins** [reply]
The answer is in your screenshot

**Reply by blacelle** [reply]
Dumb me. Any recommendation not to require CODE_OWNER review for renovate PRs? Thanks (i.e. is adding @renovate or @renovate-bot, or @app/renovate supposedly the way to proceed?)

edit: I fear this is not possible https://github.community/t/is-it-possible-to-add-github-apps-bot-to-codeowners/3146

edit2: I may be feasible https://github.com/renovatebot/renovate-approve-bot/issues/23

Sorry for not having spot this earlier.

**Reply by rarkins** [reply]
If any codeowners review is requested (automatically by github) then there's no way we know to bypass it, including with the approve bot. If you simply need *any* review in order to merge, then the approve bot you found should be sufficient.
